### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by phpcodex**

### CODE REVIEW

File: pmblocklist.php [hunk 1]
```diff
@@ -64,7 +63,7 @@
 		MiscFunctions::exitPage(strlen($void)." error(s) occured:<br />".$echo, true);
 	}
 	unset($rec, $user, $ck, $k, $void, $new, $f, $i);
-} elseif($_GET["action"] == "unblock") {
+} elseif($_POST["action"] == "unblock") {
```
Comment (halamix2) [inline pmblocklist.php:66]: Every other `if` checks for $_GET, maybe we should change the form to GET instead of changing if to POST?

**Comment by halamix2** [comment]
fixes #39 and fixes #42

**Review by halamix2 — CHANGES_REQUESTED** [review]

File: pmblocklist.php (hunk 1 above)
Comment (halamix2) [reply pmblocklist.php:66]: We'll stick to POST for now, since it can be longer than GET, and less ugly when unblocking many users

**Review by halamix2 — APPROVED** [review]

**Comment by blackcrack** [comment]
Hi :)
https://stackoverflow.com/questions/16517097/php-get-or-post-security
https://stackoverflow.com/questions/5287145/php-security-using-post-instead-of-get-to-protect-against-xsrf
http://cowbelljs.blogspot.de/2011/12/get-and-post-security.html

the answers was give by the past ;) 
i read also the oracle Internet, because the Oracle have many more experience .. and at end take/ask i am my feeling.. but before i suck up the information from the internet to become the need experiences and at end it is my feeling.
So, "post" it is sending something, "get" it is to become something .. and at end it is the security and the feeling..


best regards
blacky

**Comment by halamix2** [comment]
Fixes #13 as well, I've overlooked it

**Pull request merged**